pkg/virt-controller/services/template_test.go

```diff
@@ -1916,47 +1917,42 @@ var _ = Describe("Template", func() {
 		})
 
 		Context("with sriov interface", func() {
+			getVMIWithSriovInterface := func(name, uid string) *v1.VirtualMachineInstance {
```

Comment:
Please do not use a lambda is it is not needed. It makes it harder to figure out if an outer variable was used. Try to make it a regular function.

Regarding the name, the "get" is always overloaded. In this case `new` seems to fit better.

Reply:
done